x-pack/plugins/maps/public/trigger_actions/synchronize_movement_action.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { createReactOverlays } from '@kbn/kibana-react-plugin/public';
+import { Embeddable, EmbeddableInput } from '@kbn/embeddable-plugin/public';
+import { createAction } from '@kbn/ui-actions-plugin/public';
+import type { Embeddable as LensEmbeddable } from '@kbn/lens-plugin/public';
+import { MAP_SAVED_OBJECT_TYPE } from '../../common/constants';
+import { getCore } from '../kibana_services';
+
+export const SYNCHRONIZE_MOVEMENT_ACTION = 'SYNCHRONIZE_MOVEMENT_ACTION';
+
+interface SynchronizeMovementActionContext {
+  embeddable: Embeddable<EmbeddableInput>;
+}
+
+export const synchronizeMovementAction = createAction<SynchronizeMovementActionContext>({
+  id: SYNCHRONIZE_MOVEMENT_ACTION,
+  type: SYNCHRONIZE_MOVEMENT_ACTION,
+  order: 21,
+  getDisplayName: ({ embeddable }: SynchronizeMovementActionContext) => {
+    return i18n.translate('xpack.maps.synchronizeMovementAction.title', {
+      defaultMessage: 'Synchronize map movement',
+    });
+  },
+  getDisplayNameTooltip: () => {
+    return i18n.translate('xpack.maps.synchronizeMovementAction.tooltipContent', {
+      defaultMessage: 'Configure movement between map panels',
```

Comment:
Not sure the word "panel" is needed here.  Also, I think it might be nice to explain what "movement" means.  Here is an attempt:

> Synchronize the maps on a dashboard, so that if you zoom and pan in one map, the movement is reflected in other maps.

Reply:
Can not use "dashboard" since message is also displayed in canvas. How about "Synchronize maps, so that if you zoom and pan in one map, the movement is reflected in other maps"?